scam-detector-py/src/agent.py

```diff
@@ -1361,9 +1474,20 @@ def handle_block(block_event: forta_agent.block_event.BlockEvent) -> list:
             persist_state()
             logging.info(f"{BOT_VERSION}: Persisted state")
             
-            
-            
-        
+        # Create reactive FP findings
```

Comment:
here you also need to look at the obtain_all_fps function, because an address could have created deriverative FPs, which also need to be removed from the system.

Reply:
Will add this 👍